lib/WebpackOptionsDefaulter.js

```diff
@@ -61,6 +61,7 @@ class WebpackOptionsDefaulter extends OptionsDefaulter {
 		this.set("output.hotUpdateChunkFilename", "[id].[hash].hot-update.js");
 		this.set("output.hotUpdateMainFilename", "[hash].hot-update.json");
 		this.set("output.crossOriginLoading", false);
+		this.set("output.jsonpScriptType", "text/javascript");
```

Comment:
`type="text/javascript"` is [the implicit default](https://mathiasbynens.be/notes/html5-levels#type-attributes) for all `<script>`s, and should thus be omitted from the output.  Ref. #5221. Does anyone want to submit a follow-up PR?

Reply:
i.e. we should either output `<script type="module" …>` or just `<script>`.